antwar.config.js

```diff
@@ -0,0 +1,133 @@
+var path = require('path');
+var prevnextPlugin = require('antwar-prevnext-plugin');
+var markdown = require('./utilities/markdown');
+var highlight = require('./utilities/highlight');
+
+module.exports = {
+  //assets: [] // custom assets to copy into the build
+  output: 'build',
+  title: 'webpack',
+  keywords: ['webpack', 'javascript', 'web development', 'programming'],
+  deploy: {
+    branch: 'gh-pages'
+  },
+  pageTitle: function(config, pageTitle) {
+    var siteName = config.name;
+
+    if(pageTitle === 'index') {
+      return siteName;
+    }
+
+    return siteName + ' - ' + pageTitle;
+  },
+  plugins: [
+    prevnextPlugin()
+  ],
+  layout: function() {
+    return require('./layouts/Body.jsx')
+  },
+  style: function() {
+    require('./node_modules/normalize.css/normalize.css');
+    require('./styles/custom.scss');
+    require('./styles/min.css'); // http://mincss.com/. @Greg, drop this?
+    require('./styles/prism.css');
+    require('./styles/fontello.css');
+    require('./styles/fontello-codes.css');
+    require('./styles/fontello-embedded.css');
+    require('react-ghfork/gh-fork-ribbon.css');
+  },
+  paths: {
+    '/': root(
+      function() {
+        return require.context(
+          'json!yaml-frontmatter!./content',
+          false,
+          /^\.\/.*\.md$/
+        );
+      }
+    ),
+    concepts: section(
+      'Concepts',
+      function() {
+        return require.context(
+          'json!yaml-frontmatter!./content/concepts',
+          false,
+          /^\.\/.*\.md$/
+        );
+      }
+    ),
+    'how-to': section(
+      'How to',
+      function() {
+        return require.context(
+          'json!yaml-frontmatter!./content/how-to',
+          false,
+          /^\.\/.*\.md$/
+        );
+      }
+    ),
+    'api': section(
+      'API',
+      function() {
+        return require.context(
+          'json!yaml-frontmatter!./content/api',
+          false,
+          /^\.\/.*\.md$/
+        );
+      }
+    )
+  }
+}
+
+function root(contentCb) {
+  return {
+    title: 'Webpack',
+    path: function() {
+      return contentCb();
+    },
+    processPage: {
+      url: function(o) {
+        return o.sectionName + '/' + o.fileName.split('.')[0]
+      },
+      content: function(o) {
+        var content = o.file.__content.split('\n').slice(1).join('\n')
```

Comment:
You mean `root(contentCb)`? The system relies on `require.context` for figuring out what content to load so I need to pass the context there somehow.

I simplified it further and added comments to clear it up.